src/components/Header/Header.scss

```diff
@@ -0,0 +1,87 @@
+@import '../../variables.scss';
+@import '../../mixins.scss';
+
+.header {
+    width: 100%;
+    height: 300px;
+    position: relative;
+
+    .header_cover {
+        width: 100%;
+        height: 100%;
+        background-image: url(moviecover.jpeg);
+        filter: blur(2px);
+        z-index: -1;
+
+        @include absoluteToTopLeft
```

Comment:
A general advice on mixins: Always **include them at the beginning**, so

- you might have the chance to override parts of it later on
- easier to grab at first sight, what's being included in the current style def, when all the includes are grouped at the beginning